src/containers/DataTable/index.js

```diff
@@ -36,6 +36,14 @@ export function DataTable ({
 		}, 500);
 	}, [filterValue]);
 
+	const downloadHandler = () => {
+		const list = onDownload();
+		let csv = headers.map(header => header.name).join(',') + "\n" + 
+			list.map(item => headers.map(header => item[header.key]).join(',')).join('\n');
+		let blob = new Blob([csv], {type: "text/csv;charset=utf-8"})
+		saveAs(blob, 'table.csv');
```

Comment:
@titkov49 Can be the default name of the table dynamic? e.g. the name of the product based on the `brand_title`, name of the container, where datatable component is used and/or the current datetime? Example:

```
saveAs(blob, `${datetime}_${Config.get('brand_title')}_${containerName}_table.csv');
```

or something with similar composition/order.

Reply:
Added changes